platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectTypes.java

```diff
@@ -10,29 +10,27 @@
  *******************************************************************************/
 package org.eclipse.che.api.project.server;
 
+import org.eclipse.che.api.core.NotFoundException;
 import org.eclipse.che.api.core.ServerException;
 import org.eclipse.che.api.core.model.project.type.Attribute;
-import org.eclipse.che.api.core.model.project.type.ProjectType;
+import org.eclipse.che.api.project.server.type.ProjectTypeDef;
 import org.eclipse.che.api.project.shared.dto.SourceEstimation;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
invalid practice to have wildcard import. There are several other places in the changelog where it is the case
https://github.com/codenvy/codenvy-dev-resources/wiki/Source-file#imports